### DISCUSSION TITLE
Suppress printing of the test name on test execution

### DISCUSSION THREAD

**Discussion by JanValiVales** [pytest-dev/pytest#9093] [General]
Just for demonstration purposes let's suppose I have a very simple test in a module _test_simple.py_:

```python
import pytest

def test_example():
    print("test started")
    assert True`
```

When I run the test simply by:

```
pytest -s test_simple.py
```

I get an output:

```
`test_simple.py::test_example test started
PASSED

=============== 1 passed in 0.01s ===============
```

Is there a way how to get rid of the test name, i. e. _test_simple.py::test_example_ from the output? I was thinking about **pytest_runtest_logstart** in _conftest.py_, but I don't know how to use it to suppress the printing. Thank you.

**Comment by nicoddemus** [comment]
`-q` (quiet) will suppress the test name. Does that output what you want?

**Reply by JanValiVales** [reply]
Using 'quiet', i.e. `pytest -s -q test_simple.py` does not solve it and I still get the same output.
What I want is this:

`test started
PASSED

=============== 1 passed in 0.01s ===============

**Reply by RonnyPfannschmidt** [reply]
Based on the original code, im certain details are missing as verbose output happens

**Reply by JanValiVales** [reply]
Indeed, you are right, @RonnyPfannschmidt 
Sorry, that I missed it. There is a file pytest.ini with this content:
`[pytest]
log_cli = 1
log_cli_level = INFO
log_cli_format = %(asctime)s.%(msecs)03d - [%(levelname)8s] >>> %(message)s ... %(filename)s: %(lineno)s
log_cli_date_format=%Y-%m-%d %H:%M:%S`

When I set **log_cli = 0**, the name is not printed out. However, I loose the logs like in this example:

`import pytest
import logging

LOGGER = logging.getLogger(__name__)

def test_example():
    print("print: test started")
    LOGGER.info("logger: test started")
    assert True`

**log_cli=0** prints:
`print: test started

.
1 passed in 0.00s
`


**log_cli=1** prints:
`test_simple.py::test_example print: test started

----------------- live log call ------------------
2021-09-14 14:47:14.023 - [    INFO] >>> logger: test started ... test_simple.py: 8
PASSED

=============== 1 passed in 0.01s ================`

**Reply by RonnyPfannschmidt** [reply]
https://github.com/pytest-dev/pytest/blob/60e995d9022b18956c4b308ce64a5a8488b00f02/src/_pytest/logging.py#L684-L686

log cli triggers verbose it seems, as far asi i can tell for a good reason as well

**Reply by RonnyPfannschmidt** [reply]
i believe this could be solved if we ported terminal reporting to something like rich
